jest-preprocess.js

```diff
@@ -0,0 +1,4 @@
+const babelOptions = {
```

Comment:
added some babel dependencies and configs for using jest with gatsby acording to https://www.gatsbyjs.com/docs/how-to/testing/unit-testing/